packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -157,149 +108,79 @@ class CodeSizeBodyState extends State<CodeSizeBody>
     );
   }
 
-  void analyzeSingleSnapshot() {
-    final snapshotFile = controller.snapshotFile.value;
-    if (snapshotFile != null) {
-      controller.loadFakeTree(snapshotFile);
-    }
+  List<Widget> _buildTabViews() {
+    return [SnapshotView(), DiffView()];
```

Comment:
just put this directly where _buildTabViews() is called instead of extracting to a method